### DISCUSSION TITLE
Remove pages from menu.

### DISCUSSION THREAD

**Discussion by boberski666** [mediacms-io/mediacms#840] [Q&A]
Sorry if I am writting this i a wrong place, but I was unable to fin aswer for my question. How do I remove page links from left menu?

**Comment by CaptainTomSparta** [comment]
hello，its pretty late, but recently i started to use this repo, i love the UI. （hide menu solution i tested fine, only 1 file require amend, no need restart docker, just refresh webpage--setup Disable cache in Network from your browser）
i tested the solution in order to hide the menu: get ideas from 【mediacms/docs/admins_docs.md】( setup html page and urls ,then add extra code in _commons.js to make it work, why not hide menu from here?)

step1: find the _commons.jsfile【mediacms/static/js/_commons.js】
step2: add up code in new line: 



```
const checkElement = async selector => {
    while ( document.querySelector(selector) === null) {
      await new Promise( resolve =>  requestAnimationFrame(resolve) )
    }
    return document.querySelector(selector);
  };

// Checks that sidebar nav menu has loaded, then adds menu item.
checkElement('.nav-menu')
.then((element) => {	
        //hide the menu from parent
	var navMenu = document.querySelector('.nav-menu a[href="/about"]').closest('.nav-menu')
	navMenu.style.display = 'none';
        
        //hide individual menu
       	var aboutLink = document.querySelector('a[href="/about"]');
	var termsLink = document.querySelector('a[href="/tos"]');
	var contactLink = document.querySelector('a[href="/contact"]');
	// aboutLink.style.display = 'none';
	// termsLink.style.display = 'none';
	// contactLink.style.display = 'none';

});
```

![image](https://github.com/mediacms-io/mediacms/assets/48318680/d70aaee3-bc3c-45e1-ad0d-d1dba8800d3e)
![image](https://github.com/mediacms-io/mediacms/assets/48318680/3b23b558-1b05-4bd4-ad90-f609d47febe0)

**Comment by SaturnTeamDev** [comment]
I just came up with an alternative way of doing this after spending about three hours messing around with code. Just go to static/js/_commons.js, find the specific link you are looking to remove. Look for the keyword such as Liked or History. Use a code editor and its search feature. Once you find that block that contains all of those references, just alter the first letter of the word. Here is the example I used (biked instead of liked):
`.pages.biked.enabled&&e.push({link:f.user.biked`

This will break the push in the code to display it in the left menu.

**Comment by dnnrwttr** [comment]
Hi, is it proof for / after updating the docker container?

**Reply by SaturnTeamDev** [reply]
> Hi, is it proof for / after updating the docker container?

I have not tested my solution with a docker install.
